packages/cashier/src/Components/__tests__/cashier-container.spec.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import CashierContainer from '../cashier-container';
+
+jest.mock('@deriv/components', () => ({
+    ...jest.requireActual('@deriv/components'),
+    Loading: () => <div>Loading</div>,
+}));
+
+describe('<CashierContainer/>', () => {
+    const props = {
+        iframe_url:
```

Comment:
@likhith-deriv Could you refuse to expose the cashier url with password and secret and just pass a simple text like `test_url`

Reply:
Ok. I'll make the change 👍